packages/amplify-graphql-schema-test-library/src/__tests__/index.test.ts

```diff
@@ -0,0 +1,105 @@
+import { AuthTransformer } from '@aws-amplify/graphql-auth-transformer';
+import { DefaultValueTransformer } from '@aws-amplify/graphql-default-value-transformer';
+import { FunctionTransformer } from '@aws-amplify/graphql-function-transformer';
+import { HttpTransformer } from '@aws-amplify/graphql-http-transformer';
+import { IndexTransformer, PrimaryKeyTransformer } from '@aws-amplify/graphql-index-transformer';
+import { ModelTransformer } from '@aws-amplify/graphql-model-transformer';
+import { PredictionsTransformer } from '@aws-amplify/graphql-predictions-transformer';
+import {
+  BelongsToTransformer,
+  HasManyTransformer,
+  HasOneTransformer,
+  ManyToManyTransformer,
+} from '@aws-amplify/graphql-relational-transformer';
+import { SearchableModelTransformer } from '@aws-amplify/graphql-searchable-transformer';
+import { ConflictHandlerType, GraphQLTransform, GraphQLTransformOptions } from '@aws-amplify/graphql-transformer-core';
+import { TransformerPluginProvider } from '@aws-amplify/graphql-transformer-interfaces';
+import { schemas, TransformerPlatform, TransformerSchema, TransformerVersion } from '..';
+type Writeable<T> = { -readonly [P in keyof T]: T[P] };
+const defaultDataStoreConfig = {
+  resolverConfig: {
+    project: {
+      ConflictDetection: 'VERSION',
+      ConflictHandler: ConflictHandlerType.AUTOMERGE,
+    },
+  },
+} as any;
+
+test('v2 schemas pass or fail as expected', () => {
+  for (const [name, schema] of Object.entries(schemas)) {
```

Comment:
I'm forgetting if this is straightforward in this case (it was a pain in cypress) but could you flip these two lines, so you generate a named test per-schema `${name} v2 schema passes or fails as expected`? May help with readability and quickly identifying which specific cases are failing on a breaking change.

Reply:
Good call. It seems to work nicely with Jest.